CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+### [v4.8 _(Mar 23, 2021)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.7)
```

Comment:
We are releasing 4.8 right? 😅   (the title is 4.7 but the version is 4.8) 
```suggestion
### [v4.8 _(Apr 19, 2021)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.8)
```

Reply:
Good spot, I missed it 😅